Tasks/intersection.js

```diff
@@ -1,25 +1,16 @@
 // Find an intersection of two dictionaries
+"use strict";
 
-intersection = function intersection(object_1, object_2) {
-  first_keys = Object.keys(object_1);
-  {
-  }
-  for (attribute_name of first_keys) {
-    if (object_1[attribute_name] === object_2[attribute_name]) {
-      object_2[attribute_name] = object_1[attribute_name];
-      {
-      }
-    } else {
-      delete object_1[attribute_name];
-      {
-      }
-    }
-    {
-    }
-  }
-  {
+const intersection = (object1, object2) => {
+  const arrayOfKeys = Object.keys(object1);
+
+  for (const key of arrayOfKeys) {
+    const mutual =
+        object1[key] === object2[key]
+            ? (object2[key] = object1[key])
+            : delete object1[key];
```

Comment:
Use ternary only to branch calculations not to branch control flow, so do not put assign `=` operator and  `delete` operator inside ternary